Source/MQTTnet/Server/Internal/MqttClientStatistics.cs

```diff
@@ -73,7 +73,13 @@ public void HandleReceivedPacket(MqttPacket packet)
                 LastNonKeepAlivePacketReceivedTimestamp = LastPacketReceivedTimestamp;
             }
         }
-
+        public void ResetStatistics(){
```

Comment:
Will have a look and update...

Reply:
I will merge the pull request because it fixes the bug in the first place. We can then discuss how it is done and how to optimize it in your pull request.